tests/test_config.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import os
+import unittest
+
+import autograder.api.config
+import autograder.util.dir
+import autograder.util.dirent
+
+THIS_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+CONFIGS_DIR = os.path.join(THIS_DIR, "data", "configs")
+
+TESTS = {
+    "test_local_autograder_json_current_dir": {
+        "test_work_dir": "simple",
+        "expected_config": {
+            "user": "user@test.edulinq.org"
+        },
+        "expected_source": {
+            "user": (
+                "<default config file>:: %s"
+                % os.path.join('TEMP_DIR', 'simple', 'autograder.json')
```

Comment:
Why not a f-string on one line?

I am not saying that f-strings are required,
but you used them elsewhere so it's strange you don't use them here.